debezium-connector-postgres/pom.xml

```diff
@@ -45,6 +45,11 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
+        <dependency>
```

Comment:
We already use Jackson for dealing with JSON, so I'd prefer to use this instead of adding another dependency. I'll change it accordingly.